### DISCUSSION TITLE
v2/md-enhance/zh/guide/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#147] [Announcements]
# v2/md-enhance/zh/guide/

VuePress 的 Markdown 增强插件

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/

**Comment by zeanzai** [comment]
请问怎么配置才能开启右侧toc的3级标题呢？

**Reply by zeanzai** [reply]
我是说3级以上的标题
